docs/application/dotnet/get-started/application-filtering.md

```diff
@@ -47,22 +47,21 @@ The following table lists the available screen size features.
 | `http://tizen.org/feature/screen.size.normal.1080.1920` | Specify this key, if the application supports the 1080 x 1920 resolution on the normal screen size.<br><br>You can specify multiple `http://tizen.org/feature/screen.size.normal.*` keys, if your application supports multiple screen resolutions on the normal screen size. However, you cannot specify keys on both `screen.size.*` and `screen.size.normal.*` levels simultaneously.<br><br>If no screen size key is declared, it is assumed that the application supports only `screen.size.normal.720.1280`. To avoid this, specify at least 1 screen size key. | 2.2.1 |
 | `http://tizen.org/feature/screen.size.all` | Specify this key, if the application supports all possible current and future screen sizes and all possible current and future resolutions per screen size.<br><br>You cannot specify keys on both `screen.size.*` and `screen.size.normal.*` levels simultaneously. If you do, only the most specific resolution key is considered and the less specific resolution keys are ignored. For example, if you specify both `http://tizen.org/feature/screen.size.all` and `http://tizen.org/feature/screen.size.normal.320.480` keys, only the `http://tizen.org/feature/screen.size.normal.320.480` key is applied.<br><br>If no screen size key is declared, it is assumed that the application supports only `screen.size.normal.720.1280`. To avoid this, specify at least 1 screen size key. | 2.2.1 |
 
-### Feature Hierarchy
+### Feature hierarchy
 
 The feature keys have a hierarchy. For example, consider the `http://tizen.org/feature/location`, `http://tizen.org/feature/location.gps`, and `http://tizen.org/feature/location.wps` features:
 
--   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the
-    application on the Tizen Store application list.
+-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on Tizen Store application list.
```

Comment:
Minor correction to address a grammatical issue here.
```suggestion
-   If the feature list includes the `http://tizen.org/feature/location.gps` feature, only a device which has the `http://tizen.org/feature/location.gps` feature can show the application on Tizen Store's application list.
```

Reply:
Thank you. Your suggestion was applied.